statusboard/src/pages/Projects.test.js

```diff
@@ -1,27 +1,41 @@
 import { rest } from 'msw';
 import { setupServer } from 'msw/node';
 import React from 'react';
+import { StaticRouter } from 'react-router-dom';
 
 import { render, screen } from '@testing-library/react';
 
-import Projects from './Projects';
+import Projects from './Projects/Projects';
 import { SAMPLE_BRIGADE } from '../utils/utils.test';
+import { BrigadeDataContextProvider } from '../contexts/BrigadeDataContext';
 
 const fakeServer = setupServer(
   rest.get('/api/data.json', (req, res, ctx) => {
     return res(ctx.delay(), ctx.json([SAMPLE_BRIGADE]));
-  })
+  }),
```

Comment:
This trailing comma may cause an error.

Reply:
Good catch! I'll need need to revise which ES version should be used in https://github.com/codeforamerica/brigade-project-index-statusboard/pull/83